src/app/app.component.ts

```diff
@@ -116,9 +116,11 @@ export class AppComponent extends SubscriptionManager implements OnInit {
         }
 
         // load landing data if on the landing page
-        if (this.location.path().split('?')[0] === '') {
+        const path = this.location.path();
+        if (path.split('?')[0] === '' || path.split('/country')[0] === '') {
```

Comment:
A class constant on `app.component.ts` wouldn't work because of self-referencing imports, so it would have to go elsewhere ("`app-urls.ts`"?), and that would need to be imported by the `Country` and `Header` components and then declared public in the `.ts` of to be accessible by their respective templates.  The same would need done for the other urls for consistency.

Personally I find code easier to read when variables (that never change) don't have to be looked up... but best practice would dictate that it should be done in case we ever renamed the country page.
 
It would take a bit of refactoring and thinking about, and goes beyond the scope of this ticket.  Can we create another one for this?

Reply:
Agreed. Sounds like a separate effort.